src/popup/components/form/FormTextarea.vue

```diff
@@ -45,17 +49,41 @@ export default {
     size: {
       type: String,
       default: null,
-      validator: (val) => SIZES.includes(val),
+      validator: (val: string) => SIZES.includes(val),
     },
+    autoHeight: Boolean,
   },
-  methods: {
-    handleEnterClick(event) {
-      if (!this.enterSubmit) return;
-      this.$emit('submit');
-      event.preventDefault();
-    },
+  setup(props, { emit }) {
+    const textarea = ref<HTMLTextAreaElement>();
+    const height = ref('auto');
+    function handleInput(event: InputEvent) {
+      const { value } = event.target as HTMLInputElement;
+      if (props.autoHeight && textarea.value) {
+        height.value = 'auto';
+        nextTick(() => {
+          const { scrollHeight, clientHeight } = textarea.value as HTMLTextAreaElement;
```

Comment:
Instead of using `as HTMLTextAreaElement` you probably can add `!` at the end of `textarea.value` to indicate it's defined.